elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryThread.java

```diff
@@ -49,10 +53,13 @@ public AsyncAPIResult call() throws URISyntaxException, NoHttpResponseException
         ElideResponse response = null;
         log.debug("AsyncQuery Object from request: {}", this);
         UUID requestUUID = UUID.fromString(queryObj.getRequestId());
+        Map<String, List<String>> bypassCacheMap = new HashMap<String, List<String>>();
+        bypassCacheMap.put("bypasscache", Arrays.asList(String.valueOf(service.isBypassCache())));
+
         if (queryObj.getQueryType().equals(QueryType.JSONAPI_V1_0)) {
-            response = executeJsonApiRequest(service.getElide(), user, apiVersion, requestUUID);
+            response = executeJsonApiRequest(service.getElide(), user, apiVersion, requestUUID, bypassCacheMap);
```

Comment:
Instead of naming the variable bypassCacheMap, call it headers.  It is a better reflection of what the map represents.